addons/l10n_rs_edi/views/account_move.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="view_move_form_inherit" model="ir.ui.view">
```

Comment:
```suggestion
    <record id="view_move_form" model="ir.ui.view">
```
or
```suggestion
    <record id="view_move_form_inherit_l10n_rs_edi" model="ir.ui.view">
```
but it will be redundant with the module's name